example/Example.hs

```diff
@@ -1,196 +1,235 @@
-{-# LANGUAGE ConstraintKinds     #-}
-{-# LANGUAGE DataKinds           #-}
-{-# LANGUAGE FlexibleContexts    #-}
-{-# LANGUAGE LambdaCase          #-}
-{-# LANGUAGE OverloadedStrings   #-}
-{-# LANGUAGE ScopedTypeVariables #-}
-
```

Comment:
As an aside: you might want to actually leave the extensions here; not everybody uses default extensions, and it's convenient to have the one you need listed in this fashion in the example module, so they can copy/paste it.

Reply:
Fair enough, done.